cristin-project/src/main/java/no/unit/nva/cristin/projects/common/CristinQuery.java

```diff
@@ -238,24 +240,21 @@ private String toCristinQueryValue(Map.Entry<QueryParameterKey, String> entry) {
             return Arrays.stream(entry.getValue().split(","))
                        .collect(Collectors.joining("&" + key));
         }
-        var value = entry.getKey().isEncode()
-                        ? encodeUTF(entry.getValue())
-                        : entry.getValue();
-
+        var value = entry.getValue();
         if (entry.getKey().equals(STATUS)) {
             return ProjectStatus.valueOf(value).getCristinStatus();
         }
         return entry.getKey().equals(ORGANIZATION) && entry.getValue().matches(PATTERN_IS_URL)
                    ? getUnitIdFromOrganization(value)
-                   : value;
+                   : value.replace(" ", "+");
     }
 
     private boolean nvaParameterFilter(Map.Entry<QueryParameterKey, String> entry) {
         return entry.getKey().ordinal() > IGNORE_PATH_PARAMETER_INDEX;
     }
 
     private String decodeUTF(String encoded) {
```

Comment:
decode utf or decode url?

Reply:
decode url

Reply:
There are tests that expect til to be +

Reply:
and URIWrapper does %20

Reply:
unless it already is +

Reply:
https://api.dev.nva.aws.unit.no/cristin-NP-41111-cristin-proxy-norwegian-special/project?language=nb&name=gjennom+akutt&page=1&results=5